app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+
+import { Table as NeetoUITable } from "neetoui";
+
+import Dropdown from "./Dropdown";
+import NameAndRole from "./NameRoleAndAvatar";
+import { getContactsData } from "./utils";
+
+import { noop } from "../utils";
+
+const Table = () => {
+  const [contactsData, setContactsData] = useState([]);
+
+  const CONTACTS_TABLE_COLUMNS = [
```

Comment:
Move this to a constants.js

Reply:
@yedhink , This `CONTACTS_TABLE_COLUMNS` constant has some elements that contain `jsx`, so declaring it inside a `.js` file was against `eslint` rules. True that the file in which it is currently initialised looks very bulky and ugly because of it.  Is there anything else I could do about it ?

```js
const CONTACTS_TABLE_COLUMNS = [
    {
      dataIndex: "name_and_role",
      key: "name_and_role",
      title: "Name & Role",
      width: 150,
      render: (nameAndRole, { name, role }) => (
        <NameAndRole name={name} role={role} />
      ),
    },
    {
      dataIndex: "email",
      ellipsis: {
        showTitle: false,
      },
      key: "email",
      title: "Email",
      width: 200,
    },
    {
      dataIndex: "created_at",
      key: "created_at",
      title: "Created at",
      width: 100,
    },
    {
      dataIndex: "action",
      key: "action",
      title: "",
      width: 100,
      render: () => <Dropdown />,
    },
  ];
```

Reply:
@vishnuMohanan01 I get your point.

In such cases you can move this into the utils file and rename this as a util function. It's actually better to move this into a util function. Because we might've to pass in some args and dynamically build the table column data when the app grows. You can call this util function `buildContactTableColumns`

Reply:
Sounds great! Will do.

Reply:
Fixed it with #34 .